mewarpx/mewarpx/mwxrun.py

```diff
@@ -303,6 +303,10 @@ def step(self, sim_control, interval=None):
         while sim_control.check_criteria():
             self.simulation.step(self.step_interval)
 
+        # create file to signal that simulation ran to completion
+        if self.me == 0:
+            sim_control.write_results()
```

Comment:
```suggestion
        sim_control.write_results()
```

Reply:
I'm actually going to argue against this:
- We already have the proc 0 check inside `write_results`
- It's possible a user will write a `write_func` that includes a gather over all procs for something to be calculated. With the proc 0 check here that won't work and we'll have a deadlock which will be annoying to identify.

Reply:
nevermind! My brain confused red and green ???? I have no idea. Anyway, yes, removing the =0 check is what I wanted :-D